README.md

```diff
@@ -117,9 +122,38 @@ Reconstructing dynamic 3D scenes from 2D images and generating diverse views ove
 
   [📄 Paper](https://arxiv.org/pdf/2310.10642.pdf) | [💻 Code (to be released)](https://github.com/fudan-zvg/4d-gaussian-splatting) 
```

Comment:
I think this paper is borderline in this repo. They don't use gaussian splatting as their method. They just compare their method against gs. Why do you think we should add it? Maybe we should provide a short description why we list it here?

Reply:
Definitely feel free to remove it - for me, it had the magic words "Gaussian splatting", but at this point I'm just gathering titles.